airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -938,7 +938,7 @@
 - name: Twilio
   sourceDefinitionId: b9dc6155-672e-42ea-b10d-9f1f1fb95ab1
   dockerRepository: airbyte/source-twilio
-  dockerImageTag: 0.1.4
+  dockerImageTag: 0.1.5
```

Comment:
Please do not bump the version here as we have an automation that will take care this 🙏